exchanges/exchange.go

```diff
@@ -1962,3 +1963,9 @@ func (*Base) WebsocketSubmitOrder(context.Context, *order.Submit) (*order.Submit
 func (*Base) WebsocketSubmitOrders(context.Context, []*order.Submit) (responses []*order.SubmitResponse, err error) {
 	return nil, common.ErrFunctionNotSupported
 }
+
+// MessageID returns a universally unique id using UUID V7
+// Additional parameters may be added to provide context for the message id
```

Comment:
```suggestion
```
Don't think it adds much to say you can lengthen the string

Reply:
Obviously the comment was not clear.
I'm trying to communicate to future maintainers that there's no roadblock to adding params to the method signature.
And I'm definite I want to be saying this, to steer future dev against over complicating things trying to go around this.

How about:
```
// In the future additional params may be added to method signature to provide context for the message id for overriding exchange implementations
```

Fixed https://github.com/gbjk/gocryptotrader/commit/dc4d76f6c